src/config.template.cjs

```diff
@@ -124,7 +124,7 @@ module.exports = {
 	 * cross-seed will use links of this type to inject data-based matches into your client.
 	 * Options: "symlink", "hardlink"
 	 */
```

Comment:
```suggestion
	 * cross-seed will use links of this type to inject data-based matches into your client.
	 * Options: "symlink", "hardlink"
         * Details on linking available below
         * https://www.cross-seed.org/docs/basics/faq-troubleshooting#what-linktype-should-i-use-data-based-searching
         */

Reply:
Also as a note, for the v6 docs this URL will need "data-based-searching" part removed.